horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/RunServiceImpl.java

```diff
@@ -216,7 +228,7 @@ public RunExtended getRun(int id, String token) {
               "'validationErrors', (SELECT jsonb_agg(jsonb_build_object('schemaId', schema_id, 'error', error)) FROM run_validationerrors WHERE run_id = ?1)" +
               "))::::text FROM run WHERE id = ?1", id);
       try {
-         runExtended = Util.OBJECT_MAPPER.readValue(extendedData, RunExtended.class);
+         runExtended = mapper.readValue(extendedData, RunExtended.class);
```

Comment:
`Util.OBJECT_MAPPER` registered `JavaTimeModule`. Am wondering if this change caused the Run mapping issue? If you have resolved it for the injected `mapper` we should consolidate the code base to use one object mapper than having different instances, with different behaviour. We need to resolve this global inconsistency in this PR though